libs/enigo/src/linux/nix_impl.rs

```diff
@@ -42,6 +42,37 @@ impl Enigo {
         &mut self.custom_mouse
     }
 
+    /// Clear remapped keycodes
+    pub fn tfc_clear_remapped(&mut self) {
+        if let Some(tfc) = &mut self.tfc {
+            tfc.recover_remapped_keycodes();
+        }
+    }
+
+    fn tfc_key_click(&mut self, key: Key) -> ResultType {
+        if let Some(tfc) = &mut self.tfc {
+            let res = match key {
+                Key::Layout(chr) => tfc.unicode_char(chr),
+                key => {
+                    let tfc_key: TFC_Key = match convert_to_tfc_key(key) {
+                        Some(key) => key,
+                        None => {
+                            return Err("Failed to convert key to TFC_Key".into());
+                        }
+                    };
+                    tfc.key_click(tfc_key)
+                }
+            };
+            if res.is_err() {
+                Err("Failed to click char by tfc".to_string().into())
```

Comment:
Err("Failed to click char by tfc".into())

Reply:
`Err(format!("Failed to click {:?} by tfc", key).into())`